<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGTDrawLineSet</class>
 <widget class="QWidget" name="QGTDrawLineSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGTDrawLineSet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/GTDecorationRobotTool/Resources/BaseSet_BackGround.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 X :</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DrawLineSet_Ok">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DrawLineSet_Reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>恢复默认设置</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_DrawLineSet_Cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 Y :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 Z :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 U :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 V :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线零点位置 W :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_X">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_Y">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_Z">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_U">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_V">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_InitPos_W">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线左上顶点 X :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_topLeft_X">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_topLeft_Y">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线左上顶点 Y :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_topLeft_Z">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线左上顶点 Z :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_Width">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线矩形宽度 W :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DrawLineSet_Height">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>画线矩形高度 H :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_DrawLineSet_LiftPlat_bottomPos">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>升降平台最高点 :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_DrawLineSet_LiftPlat_topPos">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>升降平台最低点 :</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGTToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">Commen/qgttoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gtdecorationrobottool.qrc"/>
 </resources>
 <connections/>
</ui>
